test/commands/deploy/start.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { MockTestOrgData, TestContext } from '@salesforce/core/lib/testSetup';
+import { stubSfCommandUx } from '@salesforce/sf-plugins-core';
+import DeployMetadata from '../../../src/commands/project/deploy/start';
+
+describe('project deploy start', () => {
+  const $$ = new TestContext();
+  const testOrg = new MockTestOrgData();
+  testOrg.isScratchOrg = true;
+  let warnStub: sinon.SinonStub;
+
+  beforeEach(async () => {
+    await $$.stubAuths(testOrg);
+    await $$.stubConfig({ 'target-org': testOrg.username });
+    warnStub = stubSfCommandUx($$.SANDBOX).warn;
+  });
+
+  afterEach(() => {
+    $$.restore();
+  });
+
+  it('should emit warning when PPDS=true', async () => {
+    $$.setConfigStubContents('SfProjectJson', {
+      contents: {
+        packageDirectories: [{ path: 'force-app', default: true }],
+        pushPackageDirectoriesSequentially: true,
+      },
+    });
+    try {
+      await DeployMetadata.run([]);
+    } catch (e) {
+      expect(warnStub.firstCall.args[0]).to.contain('pushPackageDirectoriesSequentially');
```

Comment:
let's not have tests that break if another warning happens.  Maybe looks at all the calls/args on warnStub and verify that `.some` includes the string you want?

pattern: https://github.com/salesforcecli/plugin-org/blob/024599b083cc692fb7f845a61930c5a1820c37a5/test/unit/force/org/delete.test.ts#L82